learning.py

```diff
@@ -654,15 +654,15 @@ def predict(example):
 # ______________________________________________________________________________
 
 
-def NeuralNetLearner(dataset, hidden_layer_sizes=None,
+def NeuralNetLearner(dataset, hidden_layer_sizes=[3],
                      learning_rate=0.01, epochs=100, activation = sigmoid):
     """Layered feed-forward network.
     hidden_layer_sizes: List of number of hidden units per hidden layer
     learning_rate: Learning rate of gradient descent
     epochs: Number of passes over the dataset
     """
 
-    hidden_layer_sizes = hidden_layer_sizes or [3]  # default value
+    hidden_layer_sizes = hidden_layer_sizes
```

Comment:
This line can be removed now.

Reply:
@MrDupin  I have made the changes as suggested by you. Thank You!!